src/Tempest/Console/src/Testing/ConsoleTester.php

```diff
@@ -213,11 +217,15 @@ public function assertContains(string $text, bool $ignoreLineEndings = true): se
         return $this;
     }
 
-    public function assertDoesNotContain(string $text): self
+    public function assertDoesNotContain(string $text, bool $ignoreAnsi = false): self
```

Comment:
Couldn't `asUnformattedString` also always filter out ansi codes? I haven't read further so the answer might be no :)

Reply:
It could, maybe! I updated it that way, let's see if that causes issues down the line.